source/S_Vector_Div.ptx

```diff
@@ -594,7 +594,7 @@ is created (or destroyed) at a given location.
                     <me>
                         \vG = (\frac{\partial F_3}{\partial y}-\frac{\partial F_2}{\partial z})\vi- (\frac{\partial F_3}{\partial x}-\frac{\partial F_1}{\partial z})\vj + (\frac{\partial F_2}{\partial x}-\frac{\partial F_1}{\partial y})\vk
                     </me>
-                    Show that <m>\divg(\vG)=\vec{0}</m>. Vector fields
+                    Show that <m>div(\vG)=\vec{0}</m>. Vector fields
```

Comment:
I think this is just a merge (non-)conflict with something I had cleaned up. `\div` is LaTeX for something else, so we have `\divg` in our macros to typeset div as a math operator. If you want to just updated this line in your file as well to use `\divg`, then it should vanish from the diff and we won't have issues.

Reply:
I probably should have tagged you @nelong in what I wrote above.

Reply:
I fixed the couple of places where I was not using \divg.